testsuite/forge/src/interface/chain_info.rs

```diff
@@ -142,4 +142,13 @@ impl<'t> ChainInfo<'t> {
             self.rest_api_url.clone(),
         )
     }
+
+    pub fn into_nft_public_info(self) -> NFTPublicInfo<'t> {
+        NFTPublicInfo::new(
+            self.json_rpc_url.clone(),
+            self.chain_id,
+            self.rest_api_url.clone(),
+            self.treasury_compliance_account, // TODO: change this to actual BARS account
```

Comment:
This is the account that mint bars? Would that be the sender account?

Reply:
Yes this is the account that can mint bars. In fact, it's the only account that can mint bars because BARS module checks that the mint txn is sent by @BARS address.